### ISSUE TITLE
Does OStatus work with Friendica?

### ISSUE THREAD

**Issue by AaronRetico** [pfefferle/wordpress-ostatus#6]
Sorry if this has been asked before, but I have installed OStatus and all the required plugins on to a WordPress blog, but can't seem to get a Friendica instance to recognize it. 

The Wordpress blog is in a subdirectory, so the actual blog is website.com/subfolder/. 
The Friendica instance is also in a (separate) subdirectory.

Would either of these interfere with the discovery process? Is there a specific URL that I have somehow missed, or is the discovery supposed to automagically happen when using the main Wordpress location?

Sorry again if this has been covered before. Thank you for any advice that can be given on this issue!

**Comment by poVoq** [comment]
I also can't get it to work with HubZilla (a distant Frendica fork). Maybe I am just doing something wrong? What would be the address format to add a connection?

**Comment by pfefferle** [comment]
@AaronRetico the OStatus does not really support subfolders yet. OStatus uses Webfinger for the autodiscovery and the Webfinger discovery has to be in the root folder (`https://example.com/.well-known/webfinger?resource={resource}`), but the WordPress plugin will only be able to generate the webfinger discovery in the subfolder (`https://example.com/subfolder/.well-known/webfinger?resource={resource}`) so the discovery will not work... Sorry!

**Comment by pfefferle** [comment]
@poVoq your identifiers are:

* {login name}@{user domain} (mine is for example `pfefferle@notiz.blog` or `@pfefferle@notiz.blog`)
* {user domain}/author/{author slug} (mine is for example `https://notiz.blog/author/matthias-pfefferle/`)

Try to follow one of these identifiers on HubZilla.

**Comment by poVoq** [comment]
Ok, I tried it again. Both something like @pfefferle@notiz.blog and https://notiz.blog/author/matthias-pfefferle/ works with Hubzilla 3.6. I can also comment on blog entries federated to Hubzilla. I did not test to share it, as this is on my main Hubzilla account with many connections ;) But I see no reason why it wouldn't work.

However comments made in Hubzilla (in general?) do not show up on my Wordpress page below the blog entry. Is that a missing feature, or is something going wrong there? Maybe I have to wait a bit more for the cron job to run?
